packages/web3-eth/src/web3_eth.ts

```diff
@@ -175,6 +397,22 @@ export class Web3Eth extends Web3Context<Web3EthExecutionAPI, RegisteredSubscrip
 		return rpcMethodsWrappers.getBlock(this, block, hydrated, returnFormat);
 	}
 
+	/**
+	 * @param block The {@link BlockNumberOrTag} (defaults to {@link Web3Eth.defaultBlock}) or block hash of the desired block.
```

Comment:
should add its desc

```suggestion
	 * Returns the number of transaction in a given block.
	 * 
	 * @param block The {@link BlockNumberOrTag} (defaults to {@link Web3Eth.defaultBlock}) or block hash of the desired block.
```

Reply:
The proposed description is the same as the `@returns` description. Additionally, I think the method name, `getBlockTransactionCount` is descriptive enough. The method that have descriptions are same ones that are described in 1.x's docs